### PR TITLE
Create ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58562](https://cumulocity.atlassian.net/browse/MTM-58562)
Original PR: [6206](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6206)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Navigator nodes are now correctly removed when refreshing the sub-assets grid after removing groups via API. [GRAFT][release/cd] (#6206)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Sub-assets grid refreshes correctly after removing groups via API
```

File: content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58562
+version: 1019.22.2
+---
+Navigator nodes are now correctly removed when refreshing the sub-assets grid after removing groups via API. [GRAFT][release/cd] (#6206)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT user interface, the sub-assets grid and navigator did not always update correctly after removing a group of devices via the API. This caused the navigator to still show the removed devices and made the UI inconsistent with the actual API state. With this fix, the sub-assets grid and navigator now correctly refresh after a group of devices is removed through the API, ensuring the user interface stays in sync and provides an accurate view of the current inventory.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@janhommes Please review and approve

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Group navigation tree now refreshes correctly after removing groups via Sub-Asset view
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md:3]:
```suggestion
title: Group navigation tree in navigator now refreshes correctly after removing groups via Subassets view
```

File: content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58562
+version: 1019.22.2
+---
+In the Cumulocity IoT user interface, the sub-assets grid and navigator did not always update correctly after removing a group of devices via the API. This caused the navigator to still show the removed devices and made the UI inconsistent with the actual API state. With this fix, the sub-assets grid and navigator now correctly refresh after a group of devices is removed through the API, ensuring the user interface stays in sync and provides an accurate view of the current inventory.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md:17]:
```suggestion
In the Cumulocity IoT user interface, the subassets view and the navigator did not always update correctly after removing a group of devices via the API. This caused the navigator to still show the removed devices and made the UI inconsistent with the actual API state. With this fix, the subassets view and the navigator now correctly refresh after a group of devices is removed through the API, ensuring the user interface stays in sync and provides an accurate view of the current inventory.
```

File: content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Group navigation tree in navigator now refreshes correctly after removing groups via Subassets view
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md:3]:
```suggestion
title: Groups in the navigator and subassets view now refresh correctly after removing groups via API
```

Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md:3]: @janhommes Is the title correct now? The content did not match the description below.

File: content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58562
+version: 1019.22.2
+---
+In the Cumulocity IoT user interface, the the navigator did not always update correctly after removing a group or device via the subassets view. This caused the navigator to still show the removed groups or devices and made the UI inconsistent. With this fix, the navigator now correctly refresh after a group or device is removed through the subassets view, ensuring the user interface stays in sync and provides an accurate view of the current inventory.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-22-2-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md:17]:
```suggestion
In the Cumulocity IoT user interface, the navigator did not always update correctly after removing a group or device via the subassets view. This caused the navigator to still show the removed groups or devices and made the UI inconsistent. With this fix, the navigator now correctly refreshes after a group or device has been removed through the subassets view, ensuring the user interface stays in sync and provides an accurate view of the current inventory.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**